planning/mission_planner/src/mission_planner/arrival_checker.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "arrival_checker.hpp"
+
+#include <tier4_autoware_utils/tier4_autoware_utils.hpp>
+
+#include <tf2/utils.h>
+
+// TODO(Takagi, Isamu): remove when modified goal is always published
```

Comment:
modified goal is always published with https://github.com/autowarefoundation/autoware.universe/pull/2094